examples/SampleApp/src/components/MessageSearch/MessageSearchList.tsx

```diff
@@ -72,6 +72,7 @@ export type MessageSearchListProps = {
   refreshing: boolean;
   refreshList: () => void;
   showResultCount?: boolean;
+  scrollRef: React.Ref<FlatList<any>>;
```

Comment:
I'd suspect the linter would give either a warning or an error about the `any` here, is there a more specific type it'd make sense to use here instead?

Reply:
Following are the linting errors, I get on running the code locally. If `any` would be the issue I suppose that would be a TS warning not a lint issue.
```
/home/khushal87/Projects/stream-chat-react-native/examples/SampleApp/src/components/BottomTabs.tsx
  86:13  error  Expected object keys to be in insensitive ascending order. 'canPreventDefault' should be before 'type'  sort-keys

/home/khushal87/Projects/stream-chat-react-native/examples/SampleApp/src/components/ChatScreenHeader.tsx
  2:29  warning  'Text' is defined but never used  no-unused-vars
  2:29  warning  'Text' is defined but never used  @typescript-eslint/no-unused-vars
  2:53  warning  'View' is defined but never used  no-unused-vars
  2:53  warning  'View' is defined but never used  @typescript-eslint/no-unused-vars

/home/khushal87/Projects/stream-chat-react-native/examples/SampleApp/src/components/MessageSearch/MessageSearchList.tsx
  74:33  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any

/home/khushal87/Projects/stream-chat-react-native/examples/SampleApp/src/components/UserSearch/UserSearchResults.tsx
  67:25  warning  'user' is defined but never used  no-unused-vars

/home/khushal87/Projects/stream-chat-react-native/examples/SampleApp/src/components/UserSearch/UserSearchResultsGrid.tsx
  34:13  warning  'user' is defined but never used  no-unused-vars

/home/khushal87/Projects/stream-chat-react-native/examples/SampleApp/src/context/AppContext.ts
  26:15  warning  'config' is defined but never used  no-unused-vars
  28:16  warning  'userId' is defined but never used  no-unused-vars

/home/khushal87/Projects/stream-chat-react-native/examples/SampleApp/src/hooks/useChatClient.ts
  44:11  warning  'randomSeed' is assigned a value but never used  no-unused-vars
  44:11  warning  'randomSeed' is assigned a value but never used  @typescript-eslint/no-unused-vars

/home/khushal87/Projects/stream-chat-react-native/examples/SampleApp/src/hooks/usePaginatedUsers.ts
  14:24  warning  'newText' is defined but never used  no-unused-vars
  16:16  warning  'index' is defined but never used    no-unused-vars
  26:16  warning  'user' is defined but never used     no-unused-vars

/home/khushal87/Projects/stream-chat-react-native/examples/SampleApp/src/icons/Close.tsx
  7:46  warning  'active' is defined but never used                                        no-unused-vars
  7:46  warning  'active' is defined but never used. Allowed unused args must match /^_/u  @typescript-eslint/no-unused-vars

/home/khushal87/Projects/stream-chat-react-native/examples/SampleApp/src/icons/RightArrow.tsx
  7:51  warning  'active' is defined but never used                                        no-unused-vars
  7:51  warning  'active' is defined but never used. Allowed unused args must match /^_/u  @typescript-eslint/no-unused-vars

/home/khushal87/Projects/stream-chat-react-native/examples/SampleApp/src/screens/AdvancedUserSelectorScreen.tsx
  53:18  warning  'text' is defined but never used  no-unused-vars

/home/khushal87/Projects/stream-chat-react-native/examples/SampleApp/src/screens/ChannelListScreen.tsx
   79:37  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
  111:55  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any

/home/khushal87/Projects/stream-chat-react-native/examples/SampleApp/src/screens/MentionsScreen.tsx
  65:37  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any

/home/khushal87/Projects/stream-chat-react-native/examples/SampleApp/src/screens/NewDirectMessagingScreen.tsx
    1:17  warning  'useCallback' is defined but never used                no-unused-vars
    1:17  warning  'useCallback' is defined but never used                @typescript-eslint/no-unused-vars
    2:10  warning  'Alert' is defined but never used                      no-unused-vars
    2:10  warning  'Alert' is defined but never used                      @typescript-eslint/no-unused-vars
    9:3   warning  'SendButton' is defined but never used                 no-unused-vars
    9:3   warning  'SendButton' is defined but never used                 @typescript-eslint/no-unused-vars
   10:3   warning  'SendButtonProps' is defined but never used            no-unused-vars
   10:3   warning  'SendButtonProps' is defined but never used            @typescript-eslint/no-unused-vars
  166:10  warning  'messageInputText' is assigned a value but never used  no-unused-vars
  166:10  warning  'messageInputText' is assigned a value but never used  @typescript-eslint/no-unused-vars

✖ 33 problems (1 error, 32 warnings)
```

Reply:
```
/home/khushal87/Projects/stream-chat-react-native/examples/SampleApp/src/components/MessageSearch/MessageSearchList.tsx
  74:33  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
```

The linter picked it up as well

Reply:
Ohh yes. Let me see how I can fix it.

Reply:
You can ignore existing warnings or ts error for now. We can fix them as a separate task

Reply:
But don't introduce any new once haha